components/admin_console/workspace-optimization/dashboard.data.tsx

```diff
@@ -25,6 +25,7 @@ type DataModel = {
     [key: string]: {
         title: string;
         description: string;
+        hide: boolean;
```

Comment:
according to my comment below:

```suggestion
        hide?: boolean;
```

Reply:
ok